tool/tsh/common/tsh.go

```diff
@@ -3225,17 +3248,58 @@ func accessRequestForSSH(ctx context.Context, _ *CLIConf, tc *client.TeleportCli
 
 	// At this point we have exactly 1 node.
 	node := rsp.Servers[0]
+	var req types.AccessRequest
 	requestResourceIDs := []types.ResourceID{{
 		ClusterName: tc.SiteName,
 		Kind:        types.KindNode,
 		Name:        node.GetName(),
 	}}
+	switch cf.RequestMode {
+	case accessRequestModeRole:
```

Comment:
One more thing, if we're not going to handle `accessRequestModeRole` in `accessRequestForKubeCluster`, it should at least return an error